docker/my_init.d/81-recompile-assets

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# recompile assets at docker instance boot time
```

Comment:
I saw that, the reason I added that step was to be able to have a single docker image with all the assets baked in and not rely on an external mounted volume